docs/content/integrations/dagstermill/using-notebooks-with-dagster.mdx

```diff
@@ -146,47 +146,44 @@ When materialized, the `iris_kmeans_jupyter` asset will execute the notebook (`/
 
 ---
 
-## Step 3: Add a Dagster Repository and supply an I/O manager
+## Step 3: Add a Dagster Definitions object and supply an I/O manager
 
-We want to execute our Dagster asset and save the resulting notebook to a persistent location. This is called materializing the asset and to do this, we need to add the asset to a Dagster [repository](/concepts/repositories-workspaces/repositories).
+We want to execute our Dagster asset and save the resulting notebook to a persistent location. This is called materializing the asset and to do this, we need to add the asset to a Dagster [Definitions](/concepts/code-locations).
```

Comment:
```suggestion
We want to execute our Dagster asset and save the resulting notebook to a persistent location. This is called materializing the asset and to do this, we need to add the asset to a Dagster [Definitions](/concepts/code-locations) object.
```

Added `object` here, as it seemed like a word was missing.

Should this use the `PyObject` for `Definitions` instead of linking to the concept doc?